repos/system_upgrade/common/actors/peseventsscanner/libraries/pes_events_scanner.py

```diff
@@ -501,38 +503,87 @@ def remove_leapp_related_events(events):
     return res
 
 
+def include_instructions_from_transaction_configuration(rpm_tasks, transaction_configuration, installed_pkgs):
+    """
+    Extend current rpm_tasks applying data from transaction_configuration
+
+    :param PESRpmTransactionTasks rpm_tasks: Currently calculated rpm tasks based on PES data.
+    :param TransactionConfiguration transaction_configuration: Tasked configurated by user manually.
+    :param set(str) installed_pkgs: Set of distribution signed packages installed on the system.
+    :returns: updated tasks respecting configuration changes made by user
+    :rtype: PESRpmTransactionTasks
+    """
```

Comment:
@MichalHe I have updated the docstring of the function so it's easier to go through just inside the scope of the function without the need to check what are the input arguments.